docs/advanced-topics/using-callbacks.md

```diff
@@ -58,6 +58,24 @@ If a different model should be used for the `index_data`, you can set this via t
 For an example, see {doc}`/notebooks/text_to_image`.
 ```
 
+### Show evaluation metrics
+
+During the fine-tuning process, the evaluation metrics are displayed in the logs, which you can retrieve via the {func}`~Run.logs()` function.
+After running the fine-tuning job has finished, the evaluation metrics can be retrieved from the cloud by calling the {func}`~Run.metrics()` function.
+This function returns a JSON object with the metrics before and after fine-tuning.
+Alternatively, you can also retrieve the metrics via the {func}`~Run.display_metrics()` function, which prints the evaluation results in the form of a table to the console.
+
+![Evaluation Metrics](https://user-images.githubusercontent.com/6599259/224283786-5803fb8e-6d40-4eb7-b1d2-ae91a24648e7.png)
+
+### Display example results
+
+If you want to compare the top-k retrieval results before and after fine-tuning, you can set the `gather_examples` parameter of the evaluation callback to `True`.
+In this case, the evaluation callback will store the top-k results for each query document before and after fine-tuning.
+You can retrieve them with the {func}`~Run.example_results()` function. 
+Alternatively, you can use the {func}`~Run.display_metrics()` function to display a table of the Top-K results before and after fine-tuning to the console.

```

Comment:
Maybe explain what K is in this context and how to set it

Reply:
ok I mention what Top-K means, how to set `k` is explained in the developer documentation which you get when clicking on the link